### DISCUSSION TITLE
How to stop the models download?

### DISCUSSION THREAD

**Discussion by PolymathAli** [lllyasviel/Fooocus#1605] [Q&A] [answered]
i can't seem to stop juggernut download even after config editing and looking out for the comma and all, HELP!

**Comment by mashb1t** [comment]
This could have multiple reasons:

- Config: Please make sure to use the same default_model name as the key in checkpoint_downloads (reference: https://github.com/lllyasviel/Fooocus/pull/1167). You can always revert your changes to the known working ones of this repository (just git reset or download content again and replace it)
- Storage: make sure you have enough storage available on your host. If Fooocus fails to download a model, it can not recognize it as successfully downloaded after restarting the application.

Are you using a cloud instance or a local computer? Are there any logs in the terminal?
Hope this helps!

**Comment by thiner** [comment]
One scenario I used to experience is that the model file name was modified by Fooocus, added `.corrupted` as suffix. You may need to double check your model file name, rename the model file if it contains the suffix.

**Comment by RichterL** [comment]
if you use subdirectories to organize your checkpoints - fooocus searches only the root of the folder structure and won't find your model moved to a subdirectory. You can create a simlink in the root or move the model there to stop the downloads. Would be nice to be able to skip downloads altogether through config though.

**Reply by RichterL** [reply, marked as answer]
I take it back, there is of course an option to disable downloads.

```
{
    "checkpoint_downloads": {},
    "lora_downloads": {},
    "embeddings_downloads": {}
}
```

And to get around the error when it can't load the default model as it's in subdirectory

```
{
    "default_model": "subdirectory\\default_model.safetensors"
}
```
